tests/UnionTest.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace Psalm\Tests;
+
+use Psalm\Internal\Type\TypeCombiner;
+use Psalm\Type\Atomic;
+use Psalm\Type\Union;
+
+class UnionTest extends TestCase
+{
+    private const KNOWN_ATOMICS = [
```

Comment:
never-return => never
positive-int => `int<1, max>`
html-escaped-string => delete
false-y => delete
empty => delete

I think it should be better after that